astropy/visualization/tests/test_lupton_rgb.py

```diff
@@ -183,7 +183,7 @@ def test_make_rgb(self):
             lupton_rgb.make_lupton_rgb(
                 red, green, blue, self.min_, self.stretch_, self.Q, filename=temp
             )
-            assert os.path.exists(temp.name)
+            assert Path(temp.name).exists()
```

Comment:
This test can be lightly refactored to use `tmp_path` instead of explicitly making a named temp file.

https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
I will do the refactor for test in a new PR :)

Reply:
SGTM